.changeset/four-trains-raise.md

```diff
@@ -0,0 +1,20 @@
+---
+"astro-integration-kit": patch
+---
+
+Deprecates the `addDts` utility, it will be removed in a future minor release. Bump your Astro peer dependency to ^4.14.0 and use [injectTypes](https://docs.astro.build/en/reference/integrations-reference/#injecttypes-options):
```

Comment:
```suggestion
Deprecates the `addDts` utility, it will be removed in a future minor release. You can anticipate this change by bumping your Astro peer dependency to ^4.14.0 and using [injectTypes](https://docs.astro.build/en/reference/integrations-reference/#injecttypes-options):
```